apps/src/blockly/types.ts

```diff
@@ -146,6 +151,8 @@ export type GoogleBlocklyInstance = typeof GoogleBlockly;
 export interface ExtendedBlockSvg extends BlockSvg {
   isVisible: () => boolean;
   isUserVisible: () => boolean;
+  // used for function blocks
+  functionalSvg_?: BlockSvgFrame;
```

Comment:
do we need the underscore at the end here? similar to above - still learning 😃

Reply:
Underscore at the end is just a convention. This is a variable name we are already using so I didn't want to change it. Blockly had a lot of variables with `_` at the end, and I think we adopted that in some places. But they are moving away from that syntax, and I think we could too.